docs/development/setup-local.md

```diff
@@ -0,0 +1,254 @@
+so that Cargo can properly build:
+```sh
+; rm -rf packages/attune-ee
+```
+
+If you're an employee, you'll also want to clone the git submodules:
+```sh
+; git submodule init
+; git submodule update
+```
+
+## Install prerequisites
+
+You'll need the following tools installed and configured:
+- Docker ([desktop](https://docs.docker.com/get-started/get-docker/), [engine](https://docs.docker.com/engine/install/))
+- PostgreSQL ([Docker](https://hub.docker.com/_/postgres), [macOS](https://postgresapp.com/))
+- Minio ([quickstart](https://charts.min.io/))
+- Rust ([rustup](https://rustup.rs/), [mise](https://mise.jdx.dev/lang/rust.html))
+- Go ([go.dev](https://go.dev/), [mise](https://mise.jdx.dev/lang/go.html))
+- NPM and Node ([nvm](https://github.com/nvm-sh/nvm), [volta](https://volta.sh/), [mise](https://mise.jdx.dev/lang/node.html))
+- GPG (install using your package manager)
+
+## Build the Attune CLI
+
+We'll need to use the Attune CLI as part of these steps.
+
+Build and install it:
+```sh
+; cd cli
+; go install ./...
+```
+
+## Configure local database
+
+This service requires PostgreSQL.
+
+If you're running in Docker, you can configure the options directly:
+```sh
+# These match the defaults in the `.env.example` file.
+# You can use different values here, just make sure to modify your `.env` file accordingly.
+; docker run --name attune-postgres \
+-e POSTGRES_USER=attune \
+-e POSTGRES_PASSWORD=attune \
+-d postgres
+```
+
+If you're running natively, once it's running configure it with user credentials and a database:
+```sh
+# Log in to your postgres service.
+; psql
+
+# These match the defaults in the `.env.example` file.
+# You can use different values here, just make sure to modify your `.env` file accordingly.
+; create user attune with login password 'attune' superuser;
+; create database attune owner attune;
+```
+
+## Configure local object storage
+
+This services requires S3 compatible object storage,
+these instructions walk through using Minio.
+
+If running Minio natively, provide it with the same settings as you'll use in the `.env` file:
+```sh
+# These match the defaults in the `.env.example` file.
+# You can use different values here, just make sure to modify your `.env` file accordingly.
+MINIO_DOMAIN=localhost MINIO_ROOT_USER=attuneminio MINIO_ROOT_PASSWORD=attuneminio minio server ~/.minio/data
+```
+
+You can validate that minio is running with the `mc` command:
+```sh
+; mc alias set 'attune' 'http://127.0.0.1:9000' 'attuneminio' 'attuneminio'
+; mc ready attune
+```
+
+> [!WARNING]
+> Minio uses subdomains for the bucket.
+> On macOS you'll need to manually add the bucket name you configured as the subdomain:
+> ```text
+> 127.0.0.1	      localhost      attune-dev-0.localhost
+> 255.255.255.255	broadcasthost
+> ::1             localhost
+> ```
+>
+> You can validate that it is working properly by hitting its health endpoint:
+> ```sh
+> ; xh attune-dev-0.localhost:9000/minio/health/live
```

Comment:
Avoid non-standard utilities. I'd prefer `curl` here, and folks can translate into their own tool of choice (personally I prefer `http`).